### ISSUE TITLE
__init__() got an unexpected keyword argument 'loop'

### ISSUE THREAD

**Issue by muraig** [M-o-a-T/trio-ari#1]
Up:
=====================================================
----------------------------------------------------------------------
----------------------------------------------------------------------
**git clone https://github.com/M-o-a-T/aioswagger11.git
python setup.py test**
----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
--------------------------------------------------------
**python setup.py develop**

Using q:\pycharm\.virtenv\trio-project\lib\site-packages
Finished processing dependencies for aioswagger11==0.9.0.1

----------------------------------------------------------------------
----------------------------------------------------------------------
**git clone https://github.com/M-o-a-T/aioari.git
python setup.py develop**

Using q:\pycharm\.virtenv\trio-project\lib\site-packages
Finished processing dependencies for aioari==0.10.0

=====================================================
Install package.
###########################################################
File: Q:\Pycharm\Projects\Trio-Project\asyncari\examples\originate_example.py
Pycharm -> Ctrl+Shift+F10
----------------------------------------------------------------------------
Q:\Pycharm\.virtenv\Trio-Project\Scripts\python.exe Q:/Pycharm/Projects/Trio-Project/aioari/examples/originate_example.py
Exception ignored in: <function ClientSession.__del__ at 0x000001CA90D10DC0>
Traceback (most recent call last):
  File "Q:\Pycharm\.virtenv\Trio-Project\lib\site-packages\aiohttp-4.0.0a1-py3.8-win-amd64.egg\aiohttp\client.py", line 269, in __del__
    if not self.closed:
  File "Q:\Pycharm\.virtenv\Trio-Project\lib\site-packages\aiohttp-4.0.0a1-py3.8-win-amd64.egg\aiohttp\client.py", line 894, in closed
    return self._connector is None or self._connector.closed
AttributeError: _connector
Traceback (most recent call last):
  File "Q:/Pycharm/Projects/Trio-Project/aioari/examples/originate_example.py", line 119, in <module>
    client = loop.run_until_complete(init())
  File "Q:\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "Q:/Pycharm/Projects/Trio-Project/aioari/examples/originate_example.py", line 36, in init
    client = await aioari.connect(ast_url, ast_username,ast_password)
  File "q:\pycharm\projects\trio-project\aioari\aioari\__init__.py", line 24, in connect
    http_client = AsynchronousHttpClient(loop=loop, auth=ApiKeyAuthenticator(host, username + ':' + password))
  File "q:\pycharm\projects\trio-project\aioswagger11\aioswagger11\http_client.py", line 171, in __init__
    self.session = aiohttp.ClientSession(loop=loop)
TypeError: __init__() got an unexpected keyword argument 'loop'

Process finished with exit code 1

**Comment by emilyakavor** [comment]
@muraig , were you able to resolve this? If yes, how?

**Comment by emilyakavor** [comment]
For others who might come here, I realized this error was due to package versions and compatibility.
So you might want to look at the stack trace and determine which libraries are involved and try upgrading or downgrading them, keeping in mind their compatibility.
If you are lucky enough `pip install` might inform you of any incompatibilities. 

Once again, in my case, I had to upgrade to 
PyMySQL==1.1.0
aiomysql==0.2.0
